### PR TITLE
Updating login page/signup page text

### PR DESCRIPTION
**Pull request by hina-shah**
Updating the logiin/signup page text to be more descriptive.

### CODE REVIEW

File: public/index.ejs [hunk 1]
```diff
@@ -36,7 +36,7 @@
     <title><%= isDevelopment ? "HeLx UI" : "%META_TITLE%" %></title>
   </head>
   <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (frostyfan109) [inline public/index.ejs:39]: I don't understand why you changed this? The website is built on React (which is Javascript), and the content of noscript displays when Javascript is disabled. So it's more or less just an informational error message for the user that the website is unusable unless they enable JavaScript.

Feel free to improve the error message but changing this to HeLx UI would be confusing.

File: src/views/workspaces/login/login.js [hunk 2]
```diff
@@ -183,13 +183,14 @@ export const WorkspaceLoginView = withAPIReady(({
...
-                            has broad reaching benefits and can be applied in many domains.
+                            <a href="https://helxplatform.github.io/">HeLx</a> empowers researchers in domains from plant genomics to neuroscience to work with their preferred tools and apps in the cloud at scale.
+                            Its ability to empower researchers to leverage advanced analytical tools without installation or other infrastructure concerns has broad reaching benefits and can be applied in many domains.
+                            <br />&nbsp;<br />
+                            The HeLx Workspaces provide a wide array of data science tools for use by these researchers. Through the Workspaces, users explore and interact with analytic tools and data to support scientific discovery.
+                            <br />&nbsp;<br />
+                            Please sign in with one of the following options to access the HeLx Workspaces.
                         </Paragraph> : null
                     }
-                    logo={ <AppstoreOutlined /> }
```
Comment (frostyfan109) [inline src/views/workspaces/login/login.js:192]: Why'd we delete this?

File: src/views/workspaces/login/signup.js [hunk 3]
```diff
@@ -182,7 +184,6 @@ export const WorkspaceSignupView = withSocialSignupAllowed(({
                             </Paragraph>
                         </div>
                     }
-                    logo={ <AppstoreOutlined /> }
```
Comment (frostyfan109) [inline src/views/workspaces/login/signup.js:185]: Again confused why we're deleting this logo; are we not happy with how it looks or is it visual clutter or something else? I'd prefer we replace it if we just don't like the appearance.

**Review by frostyfan109 — CHANGES_REQUESTED** [review]

File: public/index.ejs (hunk 1 above)
Comment (hina-shah) [reply public/index.ejs:39]: This text was showing up on Google Search. We can change it.

File: src/views/workspaces/login/signup.js (hunk 3 above)
Comment (hina-shah) [reply src/views/workspaces/login/signup.js:185]: The hope was to make that logo to be something else and not be appstore specific.

**Review by frostyfan109 — APPROVED** [review]

**Pull request merged**